api_docs/message-formatting.md

```diff
@@ -0,0 +1,44 @@
+# Message formatting
```

Comment:
The introduction of this page could be split out as its own commit pretty easily, since it's just this file plus the `api_docs/changelog.md` changes.

Reply:
Done. In the updated changes, the new article is added in a prep commit. And the "channel" wildcard information is added to the article in the commit with those changes / the API version update.